src/org/dotnet/ci/pipelines/scm/VSTSPipelineScm.groovy

```diff
@@ -37,13 +37,13 @@ class VSTSPipelineScm implements PipelineScm {
         job.with {
             // Set up parameters for this job
             parameters {
-                // TODO: VSTS PR params
-                stringParam('sha1', '', 'Incoming sha1 parameter from the GHPRB plugin.')
-                stringParam('GitBranchOrCommit', '${sha1}', 'Git branch or commit to build.  If a branch, builds the HEAD of that branch.  If a commit, then checks out that specific commit.')
-                stringParam('GitRepoUrl', this.getGitUrl(), 'Git repo to clone.')
-                stringParam('GitRefSpec', '+refs/pull/${ghprbPullId}/*:refs/remotes/origin/pr/${ghprbPullId}/*', 'RefSpec.  WHEN SUBMITTING PRIVATE JOB FROM YOUR OWN REPO, CLEAR THIS FIELD (or it won\'t find your code)')
+                stringParam('VSTSCollectionName', this._collection, 'VSTS collection name')
+                stringParam('VSTSCredentialsId', this._credentials, 'VSTS credentials id')
+                stringParam('VSTSRepoUrl', this.getGitUrl(), 'VSTS repo to clone.')
+                stringParam('vstsRefspec', '', 'VSTS refspec')
+                stringParam('vstsBranchOrCommit', '*/master', 'VSTS commit hash')
```

Comment:
Instead of master, it should be the configured branch.  See the Github SCM for an example